src/main/resources/schema.graphqls

```diff
@@ -293,7 +293,10 @@ input SectionRegisterLabware {
 
 """A request to register one or more labware containing new sections."""
 input SectionRegisterRequest {
+    """The details of the labware being registered."""
     labware: [SectionRegisterLabware!]!
+    """The work number to associate with the registration."""
+    workNumber: String
```

Comment:
Again, I thought this is mandatory. Is it not?

Reply:
Let's go through these stories in stand-up and ask which ones are mandatory.

Reply:
I have updated the GraphQL schema.